Source/UIHeaderBlock.h

```diff
@@ -35,6 +35,8 @@ namespace reverb
         juce::TextButton irChoice;
         juce::TextButton sampleRate;
 
+        juce::AudioProcessorValueTreeState& parameters;
```

Comment:
I'm lost why is this required? Is it for the listener in the pluginEditor?

Reply:
This is required to update the ir_file_choice parameter in the static function menuCallback in PluginEditor.cpp. Because the function is static, it only has access to its received input parameters (i.e. an instance of the class UIHeaderBlock). I had to make the parameters accessible from this class to be able to change them in the function.